### PR TITLE
adding docs for masked email/phone and delete keycloak user etl jobs

### PR DESCRIPTION
**Pull request by rahul-tarento**

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @rahul-tarento,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://410.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://410.qa.docs.sunbird.org/master/developer-docs/installation/delete_keycloak_users/ <br /> - http://410.qa.docs.sunbird.org/master/developer-docs/installation/mask_user_email_phone_migration/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @rahul-tarento,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://410.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://410.qa.docs.sunbird.org/master/developer-docs/installation/delete_keycloak_users/ <br /> - http://410.qa.docs.sunbird.org/master/developer-docs/installation/mask_user_email_phone_migration/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @rahul-tarento,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://410.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://410.qa.docs.sunbird.org/master/developer-docs/installation/delete_keycloak_users/ <br /> - http://410.qa.docs.sunbird.org/master/developer-docs/installation/mask_user_email_phone_migration/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/installation/delete_keycloak_users.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Script to delete users stored in Keycloak
```
Comment (NupurSancheti) [inline developer-docs/installation/delete_keycloak_users.md]: Deleting Users from Keycloak

**Comment by NupurSancheti** [comment]
Tobe merged in 1.15 branch

**Comment by Sunitashelar** [comment]
closing the PR without merging it as there are conflicts

**Pull request closed without merging**